Pully Penelope/Assets/Scripts/EnemyMovement.cs

```diff
@@ -85,6 +96,10 @@ private void DefineState()
         {
             BehaveBeingGrabbed();
         }
+        else if (state == State.Cooldown)
+        {
+            BehaveCooldown();
```

Comment:
i would use a switch case here instead, since youre checking to see if the same thing (state) is a variety of different values

```
switch(state)
{
   case State.Standard: BehaveStandard();
       break;
    //continue...
}
```

Reply:
This makes sense, I think I will change this.